RELEASE/scripts/autoscend/auto_community_service.ash

```diff
@@ -3227,12 +3216,6 @@ void cs_initializeDay(int day)
 
 			cli_execute("auto_post_adv");
 
-			// TODO: again, this seems kind of wasteful
-			if((get_property("timesRested").to_int() < total_free_rests()) && chateaumantegna_available())
-			{
-				doRest();
-			}
```

Comment:
made an executive decision to remove since i cant imagine what it would be needed for. if you know better lmk.